docs/sources/write/shortcodes/index.md

```diff
@@ -21,7 +21,7 @@ Markdown is limited in its ability to render complex elements. Although you migh
 
 The following sections describe shortcodes available for use in Grafana Markdown files. To learn about other shortcodes, refer to the Hugo [shortcode documentation](https://gohugo.io/content-management/shortcodes/).
 
-> **Note for internal Grafana Labs contributors**: The Grafana shortcode templates are defined in the `layouts/shortcodes` folder of the website repo. To request custom shortcodes, [create an issue](https://github.com/grafana/writers-toolkit/issues).
+> **Note for internal Grafana Labs contributors**: The Grafana shortcode templates are defined in the `layouts/shortcodes` folder of the website repository. To request custom shortcodes, [create an issue](https://github.com/grafana/writers-toolkit/issues).
```

Comment:
ironically, does not use admonition shortcode.

Reply:
Resolved with https://github.com/grafana/writers-toolkit/pull/287/files/a68ca46bd455483904d867009c11c3c1fd7a6ca4#r1281553659